tests/build/fixtures/page.rs

```diff
@@ -20,14 +20,15 @@ pub fn index(config: &Config) -> String {
         contents: markdown::to_html(readme(), &config.syntax_theme).unwrap(),
         filename: "index.html".to_string(),
         is_index: true,
+        needs_js: true,
     };
-    let contents = page.build(config).unwrap();
+    let contents = page.build(config, true).unwrap();
     layout::build(config, contents, true).unwrap()
 }
 
 pub fn artifacts(config: &Config) -> String {
     let artifacts_content = artifacts::page::build(config).unwrap();
-    let page = Page::new_from_contents(artifacts_content, "artifacts.html");
-    let contents = page.build(config).unwrap();
-    layout::build(config, contents, false).unwrap()
+    let page = Page::new_from_contents(artifacts_content, "artifacts.html", true);
+    let contents = page.build(config, true).unwrap();
```

Comment:
we don't need to pass true here because this is a method called on page- this should come from the object it is called on

Reply:
also this unwrap should be a ?

Reply:
fixed